lib/service.js

```diff
@@ -7,6 +7,7 @@
 
 var thread = require('./thread-global');
 var utils = require('./utils');
+var ServiceStream = require('./service-stream');
```

Comment:
Move this to line 8 to satisfy my irrational OCD-ness :stuck_out_tongue_winking_eye: